src/plone/restapi/tests/http-examples/translated_messages_addons.resp

```diff
@@ -28,10 +28,10 @@ Content-Type: application/json
                 "available": false,
                 "hasProfile": true,
                 "installedVersion": "2.0.0",
-                "newVersion": "2.0.0",
-                "required": false
+                "newVersion": "1.2.3",
+                "required": true
```

Comment:
Why did this change the value of `required`?

Reply:
it looks like it's because the values of `installedVersion` and `newVersion` are different now that I patch the current version...

Reply:
I have reworked the patch and patched also the `upgrade_info` dict.

The code there does many things and it is hard to patch just the version numbers to get the `required` value to be false.

Reply:
@erral I'd rather not replace the `upgrade_info` with an empty dict like that. It starts to make the response less useful where it appears in documentation, because it's not realistic. Probably better to keep it as it was before.

Reply:
I have reverted the changes while I check what can I do with this.